beacon-chain/rpc/service.go

```diff
@@ -598,6 +611,49 @@ func (s *Service) Start() {
 	}()
 }
 
+func updateTrackValidatorCacheWithProposerSettings(ctx context.Context, chain blockchain.ChainInfoFetcher, settings *validatorpb.ProposerSettingsPayload, tackedValidatorCache *cache.TrackedValidatorsCache) error {
+	if settings != nil && settings.ProposerConfig != nil {
```

Comment:
Maybe I should add a warning if I'm not using the builder settings and registering that to the cache just yet.
this may go the same for graffiti